### ISSUE TITLE
Multiple post-intervention periods

### ISSUE THREAD

**Issue by FMenchetti** [FMenchetti/MachineControl#13]
So far, the functions only work with 1 post-intervention period (like in the application of the paper that inspired the package). However, to make it useful for as many users as possible, we should generalize the functions to work with multiple post-intervention periods. This may require changing the `post_period` argument with an `int_date` argument and internally defining as post-intervention period all the time points following `int.date`. While `PanelCrossValidation` shouldn't be affected by the change (it only focuses on the pre-intervention), this change will impact the bootstrap inference function.

**Comment by FMenchetti** [comment]
The ate estimation and inference has been generalized to multiple intervention periods. Only the bca boot option needs to be updated. Cate estimation and inference needs to be updated as well.

**Comment by FMenchetti** [comment]
I reopened the issue following a concern on 2 or more steps-ahead predictions. When past lags of the dependent variable are included in the model, a recursive procedure was implemented that relies on predicting Yt+2 after imputing Yt+1 with its predicted counterfactual. The code was updated and the issue can be closed. Further details on the procedure can be found on the paper Cerqua A., Letta M. & Menchetti F. (2023)
